README.md

```diff
@@ -161,6 +161,67 @@ If you want to use the deploy configuration of directory from which you are exec
 
 Note: we should NOT manually adapt code in /var/www/vhosts/mf-geoadmin3 directory
 
+
+## Building the project to int and prod (S3)
+
+Deploying a project to int or prod consists of three steps:
+
+- Building
+- Uploading to S3
+- Activating a version, as only one version may be active at time (`index.html`)
```

Comment:
http://www.gravizo.com/

Would enable us to integrat small graphs in *.md files, which would directly be shown as graphs on github.